src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -336,6 +347,20 @@
             })
             .catch(ibexa.helpers.notification.showErrorNotification);
     };
+    const toggleDraggableForFirefox = (input, eventType) => {
```

Comment:
If you want to have this method you should not pass here eventType but some flag so the result of `eventType === 'mouseenter'` to make it more reusable.
The second approach would be to remove `mouseEnterInputHandlerForFirefox`/`mouseLeaveInputHandlerForFirefox` methods and use `toggleDraggableForFirefox` directly in the eventListeners.

Reply:
With first part I would agree but about the second one I am not sure what if we would like to have specific behavriour on mouseEnter and different on mouseLeave also if we would like to have removeEventlistener it cannot be annonymous function

Reply:
There is always a way to create those wrappers in the future, but as I wrote both ways are valid.